### PR TITLE
Web-687: Content cleanup

### PR DESCRIPTION
**Pull request by spencerhunter**

### CODE REVIEW

File: pages/guides/send-money/index.mdx [hunk 1]
```diff
@@ -39,16 +39,16 @@ In this quickstart guide, you’ll learn the key concepts involved with sending
 
 ## Before you begin
```
Comment (Finholt) [inline pages/guides/send-money/index.mdx:44]: The image in this file isn't loading in

File: pages/guides/send-money/create-transfer.mdx [hunk 2]
```diff
@@ -246,6 +246,8 @@ print($transfer->status); # => "processed"
 
 That’s it! You’ve successfully transferred money to a recipient. Please continue to the Webhooks guide for information on implementing notifications for your customers about the status of the transfer.
```
Comment (Finholt) [inline pages/guides/send-money/create-transfer.mdx:250]: The image in this file isn't loading in

File: pages/guides/auth/client-credentials-flow.mdx [hunk 3]
```diff
@@ -78,9 +78,11 @@ application_token = $dwolla.auths.client
...
 A refresh token is not paired with an application access token, therefore in order to refresh authorization you'll simply request a new application access token by exchanging your client credentials (as shown above).
 
-That's it! You're ready to start [making requests](/guides/auth/using-an-access-token.html) to the [Dwolla API](https://docs.dwolla.com/) on behalf your application.
+That's it! You're ready to start [making requests](/guides/auth/using-an-access-token) to the [Dwolla API](https://docs.dwolla.com/) on behalf your application.
 
-<nav class="pager-nav">
-  <a href="./">Back: Overview</a>
-  <a href="using-an-access-token.html">Next step: Using an access token</a>
-</nav>
+<Paginaton variant="previous" variantText="PREVIOUS" href="./"></Paginaton>
```
Comment (ShreyaThapa) [inline pages/guides/auth/client-credentials-flow.mdx]: I believe Connor added an implementation for `<Pagination>` that dynamically adds the NEXT and BACK pagination for each doc in a group. https://github.com/Dwolla/dev-portal/blob/master/layouts/guides.tsx#L39
Ahhh I hate to say this after you're put in so much work into adding these to each page already! :(
We may need to create a separate cleanup ticket to remove these pagination components from the docs.

File: pages/guides/receive-money/create-a-customer.mdx [hunk 4]
```diff
@@ -99,20 +101,20 @@ print($customer); # => "https://api-sandbox.dwolla.com/customers/c7f300c0-f1ef-4
...
-<ol class="alerts">
-  <li class="alert icon-alert-alert">
-    Provide the IP address of the end-user accessing your application as the
-    `ipAddress` parameter. This enhances fraud detection and tracking.
-  </li>
-</ol>
+<AlertBar variation="info">
+  Providing the IP address of the end-user accessing your application as the
+  <code>ipAddress</code> parameter. This enhances fraud detection and tracking.
+</AlertBar>
```
Comment (Finholt) [inline pages/guides/receive-money/create-a-customer.mdx:107]: When components are used back-to-back (with no text between them) they scrunch together. In this case, it's the code block above and this alert bar. This isn’t necessarily something we have to fix ahead of the internal launch, but we should discuss and fix it ahead of public launch

**Review by Finholt — COMMENTED** [review]
A couple other things that I noticed:
- `Create a Webhook Subscription` is missing a Step 2
- A number of the guides have an `<hr>` between the content and the pagination. Is that intended?
- The titles of the docs in the `Generate an OAuth Access Token` do not follow the ‘Step X: Title’ standard that the rest of the guides establish
- The “Next Steps” links in [this file](https://github.com/Dwolla/dev-portal/blob/WEB-687/pages/guides/sandbox/index.mdx) aren’t being styled properly because they’re still being rendered inside the old `<nav>` tag

File: pages/guides/auth/client-credentials-flow.mdx (hunk 3 above)
Comment (Finholt) [reply pages/guides/auth/client-credentials-flow.mdx]: Yep, that is the case. We don't need to add the Pagination components at all within the MDX files. Sorry that I wasn't clear about that! 🙁

Comment (spencerhunter) [reply pages/guides/auth/client-credentials-flow.mdx]: Didn't spend too much time, just a lot of copy pasting ha! I can update in the next PR

Comment (ShreyaThapa) [reply pages/guides/auth/client-credentials-flow.mdx]: Sweet! Thanks :)

File: pages/guides/receive-money/create-a-customer.mdx (hunk 4 above)
Comment (ShreyaThapa) [reply pages/guides/receive-money/create-a-customer.mdx:107]: Yeah sounds good. I've seen using a `<br/>` tag in between works to an extent, but we may have to think through either adding a wrapper div around it, or something in the MDX styling that adds padding to JSX components.

**Comment by ShreyaThapa** [comment]
> * A number of the guides have an `<hr>` between the content and the pagination. Is that intended?
I noticed that too. I think it looks better without the `<hr>`, like in the mockups.

File: pages/concepts/webhook-events.mdx [hunk 5]
```diff
@@ -73,7 +81,13 @@ All webhook payloads will include an Event object which will follow the same for
...
-> **Note:** The Event topic for all events triggered for your end-users that are created as Customers will be prepended with `customer_*`. Events that represent actions being triggered on your primary Dwolla Account are not shown below. Refer to the [API docs](https://docs.dwolla.com/#events) for the complete list of Events.
+<AlertBar variation="info">
+  The Event topic for all events triggered for your end-users that are created
+  as Customers will be prepended with `customer_*`. Events that represent
+  actions being triggered on your primary Dwolla Account are not shown below.
+  Refer to the <a href="https://docs.dwolla.com/#events">API docs</a> for the
+  complete list of Events.
+</AlertBar>
 
 ### Customers
```
Comment (Finholt) [inline pages/concepts/webhook-events.mdx:92]: The final table in this file isn't rendering properly

File: pages/guides/business-verified-customer/certify-beneficial-ownership.mdx [hunk 6]
```diff
@@ -181,12 +181,13 @@ When a Customer has a `recertify` beneficial ownership status, they will have up
```
Comment (Finholt) [inline pages/guides/business-verified-customer/certify-beneficial-ownership.mdx]: In this file
> "I,____ (name of Account Admin), hereby certify, to the best of my knowledge, that the information provided above is complete and correct."

is displayed in a code example. Is that intentional?

File: pages/guides/dwolla-js/instant-account-verification/configure-and-call-function.mdx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+#### `stylesheets`
+
+Dwolla provides a list of CSS classes available for styling certain elements of the IAV flow. These elements can be easily customized to match the look and feel of your application, and are included within the `options` JavaScript object of the function dwolla.iav.start(). You can specify one or many stylesheets as a list within the `stylesheets` attribute. By default, the elements within your specified container are responsive to any change in screen size.
+
+<a
+  href="https://www.dwolla.com/dwollajs-bank-verification"
+  target="_blank"
+  class="btn secondary medium"
```
Comment (Finholt) [inline pages/guides/dwolla-js/instant-account-verification/configure-and-call-function.mdx]: This link is being styled incorrectly. We could either display it as a normal link by removing the classes here or display it as a button by importing the Button component and using that

File: pages/guides/transfer-between-users/create-verified-customer.mdx [hunk 8]
```diff
@@ -12,7 +12,7 @@ meta:
```
Comment (Finholt) [inline pages/guides/transfer-between-users/create-verified-customer.mdx:14]: All of the step numbers in the headers are off from the step numbers in the title for this guide

**Review by Finholt — COMMENTED** [review]
Round 2!

File: pages/guides/receive-money/create-a-customer.mdx (hunk 4 above)
Comment (Finholt) [reply pages/guides/receive-money/create-a-customer.mdx:107]: Adding `<br/>` tags sounds like a good quick fix though! Thanks for calling that out, Shreya!

**Review by Finholt — APPROVED** [review]
Not holding this one up anymore, but two things that we should look into with a future PR
- Style Sandbox Setup Guide better
- Fix margins between components when they're rendered back-to-back

**Pull request merged**